test/e2e/puppeteer.js

```diff
@@ -57,6 +57,7 @@ const exceptionList = [
 	'webxr_xr_ballshooter',
 
 	'webgl_worker_offscreencanvas', // in a worker, not robust
+	'webgpu_worker_offscreencanvas', // in a worker, not robust
```

Comment:
Thinking ahead? :)

Reply:
Oops I was working on it, will remove it 😁

Reply:
Solved. `webgpu_portal` probably needs a bigger idleTime so ignored for the moment.